src/codeGenerator.ts

```diff
@@ -8,7 +8,7 @@ export interface CodeFile {
 
 export function buildCodeFiles(source: string, type: string): CodeFile {
   const sourceLines = source.split("\n");
-  let fileType: string = null;
+  const fileType = type.substr(1);
```

Comment:
Why this change? This breaks the buildCodeFiles tests

Reply:
jest autorunner failed silently :angry: 

I made this change erroneously seeing another error somewhere else. Fixed